### PR TITLE
[Doc] Webhooks documentation tweaks

### PR DESCRIPTION
**Pull request by julien-c**
ready for review

### CODE REVIEW

File: docs/hub/webhooks.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-They allow you to listen for new changes on specific repos or repos belonging to particular users/organizations (not just your repos, but any repo!).
+They allow you to listen for new changes on specific repos or to all repos belonging to particular users/organizations (not just your repos, but any repo).
 
-You can create new webhooks and edit exiting ones in your webhooks [settings](https://huggingface.co/settings/webhooks):
+You can create new webhooks and edit existing ones in your webhooks [settings](https://huggingface.co/settings/webhooks):
 
 ![Settings of an individual webhook](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/webhook-settings.png)
 
-Webhooks can watch for repos updates, Pull requests, discussions, and new comments. It's even possible to create a Space to react to your webhooks!
+Webhooks can watch for repos updates, Pull requests, discussions, and new comments. If you need to host your webhook, it's of course possible to use a Space to host it.
```
Comment (mishig25) [inline docs/hub/webhooks.md:13]: for me, the previous version "It's even possible to create a Space to react to your webhooks!" was clearer

**Review by mishig25 — APPROVED** [review]
lgmt except [this nit](https://github.com/huggingface/hub-docs/pull/624#discussion_r1082468049) !

File: docs/hub/webhooks.md (hunk 1 above)
Comment (julien-c) [reply docs/hub/webhooks.md:13]: yes, feel free to revert + merge then (let's wait for @coyotte508 review maybe)

**Comment by julien-c** [comment]
please review also the related #627

File: docs/hub/webhooks.md [hunk 2]
```diff
@@ -4,13 +4,13 @@
 
 Webhooks are a foundation for MLOps related features. You can use them to auto-convert models, build community bots, or build CI/CD for your models, datasets, and Spaces.
 
-They allow you to listen for new changes on specific repos or repos belonging to particular users/organizations (not just your repos, but any repo!).
+They allow you to listen for new changes on specific repos or to all repos belonging to particular users/organizations (not just your repos, but any repo).
```
Comment (coyotte508) [inline docs/hub/webhooks.md:7]:
```suggestion
They allow you to listen for new changes on specific repos or to _all_ repos belonging to particular users/organizations.
```

File: docs/hub/webhooks.md (hunk 1 above)
Comment (coyotte508) [reply docs/hub/webhooks.md:13]: i'm split!

**Review by coyotte508 — APPROVED** [review]

File: docs/hub/webhooks.md (hunk 1 above)
Comment (julien-c) [reply docs/hub/webhooks.md:13]: ok will revert that sentence.

cc @osanseviero fyi too

Comment (julien-c) [inline docs/hub/webhooks.md:13]:
```suggestion
Webhooks can watch for repos updates, Pull requests, discussions, and new comments. It's even possible to create a Space to react to your webhooks!
```

**Comment by julien-c** [comment]
Ok, merging

**Pull request merged**